obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+- agasc_id: 798760200
+  mag_aca: 11.2
+  mag_aca_err: 0.1
+- agasc_id: 805966472
+  mag_aca: 11.703
+  mag_aca_err: 0.1
+- agasc_id: 821318496
+  mag_aca: 10.719
+  mag_aca_err: 0.1
+- agasc_id: 836372744
+  mag_aca: 11.609
+  mag_aca_err: 0.1
+- agasc_id: 836381648
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 879768704
+  mag_aca: 10.859
+  mag_aca_err: 0.1
+- agasc_id: 891159184
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 897878136
+  mag_aca: 10.75
+  mag_aca_err: 0.1
+- agasc_id: 898125104
+  mag_aca: 11.203
+  mag_aca_err: 0.1
+- agasc_id: 899950912
+  mag_aca: 11.688
+  mag_aca_err: 0.1
+- agasc_id: 901664112
+  mag_aca: 10.406
+  mag_aca_err: 0.1
+- agasc_id: 913441888
+  mag_aca: 10.031
+  mag_aca_err: 0.1
+- agasc_id: 915541320
+  mag_aca: 11.6
+  mag_aca_err: 0.1
+- agasc_id: 915939232
+  mag_aca: 12.906
+  mag_aca_err: 0.1
+- agasc_id: 962998672
+  mag_aca: 11.406
+  mag_aca_err: 0.1
+- agasc_id: 974791856
+  mag_aca: 11.375
+  mag_aca_err: 0.1
+- agasc_id: 1020796600
+  mag_aca: 11.5
+  mag_aca_err: 0.1
+- agasc_id: 1058938112
+  mag_aca: 10.906
+  mag_aca_err: 0.1
+- agasc_id: 1081092600
+  mag_aca: 11.0
+  mag_aca_err: 0.1
+- agasc_id: 1092619768
+  mag_aca: 11.141
+  mag_aca_err: 0.1
+- agasc_id: 1105592952
+  mag_aca: 10.734
+  mag_aca_err: 0.1
+- agasc_id: 1108353064
+  mag_aca: 11.094
+  mag_aca_err: 0.1
+- agasc_id: 1139171624
+  mag_aca: 10.297
+  mag_aca_err: 0.1
+- agasc_id: 1143744224
+  mag_aca: 10.6
+  mag_aca_err: 0.1
+- agasc_id: 1178221128
+  mag_aca: 10.031
+  mag_aca_err: 0.1
+obs:
+- agasc_id:
+  - 901664112
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:129:13:39:37.186
+  status: 1
+- agasc_id:
+  - 686557584
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:221:17:58:51.959
+  status: 1
+- agasc_id:
+  - 897878136
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2003:228:21:05:34.379
+  status: 1
+- agasc_id:
+  - 857375576
+  comments: Expected star position is >1000 arcsec away during the whole obs
+  mp_starcat_time: 2003:357:02:49:14.852
+  status: 1
+- agasc_id:
+  - 857375848
+  comments: Expected star position is >1000 arcsec away during the whole obs
+  mp_starcat_time: 2003:357:02:49:14.852
+  status: 1
+- agasc_id:
+  - 188482256
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2004:055:07:41:03.254
+  status: 1
+- agasc_id:
+  - 654050016
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2004:109:04:09:26.608
+  status: 1
+- agasc_id:
+  - 920782072
+  comments: NSM at start of observation
+  mp_starcat_time: 2004:200:06:56:32.527
+  status: 1
+- agasc_id:
+  - 920783352
+  comments: NSM at start of observation
+  mp_starcat_time: 2004:200:06:56:32.527
+  status: 1
+- agasc_id:
+  - 897607632
+  comments: Large magnitude variation. Known eclipsing binary
+  mp_starcat_time: 2004:212:17:56:19.266
+  status: 1
+- agasc_id:
+  - 76813072
+  comments: mag 8.1 spoiler 15 arcsec away
```

Comment:
I think the "spoiler" is the star and we have bad coordinates.  So maybe add to bad list?